reference/configuration/framework.rst

```diff
@@ -1013,6 +1016,7 @@ Each package can configure the following options:
 * :ref:`base_urls <reference-assets-base-urls>`
 * :ref:`version <reference-framework-assets-version>`
 * :ref:`version_format <reference-assets-version-format>`
+* :ref:`version_strategy <reference-assets-version-strategy>`
```

Comment:
I guess this should be `reference-framework-assets-version_strategy` instead (see the build failure on Travis CI).